lib/api.js

```diff
@@ -61,7 +61,7 @@ class JSDOM {
   }
 
   serialize() {
-    return domToHtml([idlUtils.implForWrapper(this[window]._document)]);
+    return fragmentSerialization(idlUtils.implForWrapper(this[window]._document), false);
```

Comment:
Can we use an options object instead? https://ariya.io/2011/08/hall-of-api-shame-boolean-trap